packages/documentation-framework/templates/mdx.js

```diff
@@ -84,6 +85,12 @@ const MDXChildTemplate = ({
           {' '}To learn more about the process, visit our <Link to="/get-started/about#major-release-cadence">about page</Link>.
         </InlineAlert>
       )}
+       {(template || source === 'react-template') && (
+        <InlineAlert title="Templates" variant="info">
+          {`This section showcases templates for the ${id} component. A template is a wrapped ${id} component use case that has built in logic with a streamlined API, and some limited customization.
```

Comment:
```suggestion
          {`This page showcases templates for the ${id} component. A template contains a wrapped component that has built-in logic, a streamlined API, and some limited customization.
```

What is the intended use of a template? Are they like fill-in-the-blank components? 

Having trouble understanding the second sentence well enough to suggest phrasing, but maybe that's because I am not familiar with templates as a concept

Reply:
Templates will be pre-wrapped demos basically, with a small API. Instead of a user needing to define additional boiler plate code like state and toggles (for select), they have the option to use a template which would only need some initial data and self-contain the rest of the logic to get it to function.

Updated and tweaked the second sentence a little to represent that a bit better, LMK what you think and if it needs 
 further updates 👍